packages/web-components/src/tooltip/tooltip.ts

```diff
@@ -170,7 +170,7 @@ export class Tooltip extends FASTElement {
   public showTooltip(delay: number = this.defaultDelay): void {
     setTimeout(() => {
       this.setAttribute('aria-hidden', 'false');
-      // @ts-expect-error - Baseline 2024
+      //// @ts-expect-error - Baseline 2024
```

Comment:
seems your branch was not up to date with latets ( typescript 5.2) @davatron5000 

https://github.com/microsoft/fluentui/commit/70d3d23742e1914983cde0c9a16bb36e1a35165a - after merge this broke also master pipeline. Please make sure to update your branches before merging in order to avoid these scenarios in future - github UI will present you a button once it notifies your branch is up to date.

ty

<img width="823" alt="image" src="https://github.com/user-attachments/assets/889499d1-7474-4add-b84c-0c28f7a11316">